src/plugins/validation/oas3/semantic-validators/pagination-ibm.js

```diff
@@ -28,15 +28,20 @@ module.exports.validate = function({ resolvedSpec }, config) {
 
   // Loop through all paths looking for "list" operations.
   for (const path in resolvedSpec.paths) {
+    // For now, just consider the get operation.
+    const operation = resolvedSpec.paths[path].get;
+
     // Skip any path that ends in a path parameter or does not have a "get" operation
-    if (/}$/.test(path) || !resolvedSpec.paths[path].get) {
+    if (/}$/.test(path) || !operation) {
       continue;
     }
 
-    const resp = getJsonResponse(resolvedSpec.paths[path].get);
-    const jsonResponse = resp
-      ? resolvedSpec.paths[path].get.responses[resp].content['application/json']
-      : undefined;
+    // Find first success response code
+    const resp = Object.keys(operation.responses || {}).find(code => code.startsWith('2'));
+    // Now get the json content of that response
+    const content = resp && operation.responses[resp].content;
+    const jsonResponse = content && content['application/json'];
```

Comment:
I reworked this section to put all the code inline.  I think this makes it clearer what `resp` and `jsonResponse` contain.

@dpopp07  I would like your opinion on the style I used here -- using `&&` as a form of "optional chaining".

Reply:
This looks good! The "option chaining" you're referring to is fairly common and adds a nice "safety" to JS code